src/components/v5/common/Table/Table.tsx

```diff
@@ -4,12 +4,14 @@ import {
   useReactTable,
   getCoreRowModel as libGetCoreRowModel,
 } from '@tanstack/react-table';
+import clsx from 'clsx';
 import { TableProps } from './types';
 
 const displayName = 'v5.common.Table';
 
 const Table = <T,>({
   className,
+  isError,
```

Comment:
This is not a valid approach. Check if you can use `className` for this. if not add another prop, like `tableClassName` if you really need to add a class directly to the table element

Reply:
+1